app/ViewModels/Concerns/Wallet/CanForge.php

```diff
@@ -79,4 +79,23 @@ public function isMissing(): bool
             ->filter(fn ($performance) => $performance === false)
             ->count() > 1;
     }
+
+    public function isForged(): bool
+    {
+        return collect($this->performance())->last() === true;
+    }
+
+    public function isMissedBlock(): bool
+    {
+        $performance          = collect($this->performance());
+        $missedLast           = $performance->last() === false;
+        $previousToLastForged = $performance->get($performance->count() - 2) === true;
+
+        return $missedLast && $previousToLastForged;
+    }
+
+    public function isMissed(): bool
```

Comment:
```suggestion
    public function justMissed(): bool
```

Keep those method names in line with the `Slot` class.